### ISSUE TITLE
MAPE for each epoch

### ISSUE THREAD

**Issue by soroosh-rz** [unit8co/darts#2150]
I'd like to visualize MAPE for training and validation at the end of each epoch. I wonder if it is possible to do it using the current capabilities?

**Comment by madtoinou** [comment]
Hi @soroosh-rz,

It's totally possible with current capabilities. You'll need to implement your own callback to store/retrieve the predicted and expected values before applying mape on them.

You can find an example of a callback to store the loss in this [notebook](https://unit8co.github.io/darts/userguide/torch_forecasting_models.html?highlight=callback#example-of-custom-callback-to-store-losses) and some additional information about callbacks in the pytorch lightning [documentation](https://lightning.ai/docs/pytorch/stable/extensions/callbacks.html).

**Comment by soroosh-rz** [comment]
Thanks @dennisbader. Your comment solved my problem!
